cli/index_data.py

```diff
@@ -51,6 +60,10 @@ def get_metadata_dict(task: IndexerInput) -> dict:
         **{k: v for k, v in task.dict().items() if k != "document_metadata"},
         **{f"document_{k}": v for k, v in task.document_metadata.dict().items()},
     }
+    if "document_category" in task_dict:
+        supplied_categories = task_dict["document_category"]
+        mapped_categories = [_map_old_category_to_new(c) for c in supplied_categories]
```

Comment:
`document_metadata` is a str in `IndexerInput` so I think you just need to do `task_dict["document_category"] = _map_old_category_to_new(task_dict["document_category"])`

Reply:
Good call, I had assumed that metadata was just a dict. Fixed.